### PR TITLE
Replace default values of props using global variables with computed case

### LINKED ISSUE #10829: Replace default values of props using global variables with computed case
# Description

Replace the existing default assignment with a computed case. 
TBD naming convention/logic.

## Context

In the Vue3 migration list it has been omitted [this case](https://vuejs.org/guide/components/props.html) and it needs to be handled manually, also possible before actual migration.

Screenshot 2024-04-17 at 11 55 59

### PR DESCRIPTION
**Pull request by cnotv**
### Summary
Fixes #10829

As mentioned in the issue, this is a safe migration to Vue3 with limited use cases.
Using computed values based on props using i18n default values should be sufficient to not break anywhere.

A first attempt to use the global variable `t` has been carried out, unfortunately, it's not exposed in a prop scope and cannot be used.

Finally, as you can see by the commits and logs, it's mandatory to set a default empty value regardless, to not break existing logic.

### Checklist
- [x] The PR is linked to an issue and the linked issue has a Milestone, or no issue is needed
- [x] The PR has a Milestone 
- [x] The PR template has been filled out
- [x] The PR has been self reviewed 
- [x] The PR has a reviewer assigned
- [x] The PR has automated tests or clear instructions for manual tests and the linked issue has appropriate QA labels, or tests are not needed
- [x] The PR has reviewed with UX and tested in light and dark mode, or there are no UX changes

### CODE REVIEW

File: shell/components/KeyValueView.vue [hunk 1]
```diff
@@ -70,6 +66,14 @@ export default {
   },
 
   computed: {
+    _keyLabel() {
```
Comment (codyrancher) [inline shell/components/KeyValueView.vue:69]: With the `_` prefix are you trying to denote private scope?

Stylistically I don't think I've run into this in our codebase.

**Review by codyrancher — APPROVED** [review]

File: shell/components/KeyValueView.vue (hunk 1 above)
Comment (cnotv) [reply shell/components/KeyValueView.vue:69]: Kind of. 
I just had to pair an exposed value to an internal one, so I tried to use the closest naming convention to this case.
I am aware that's usually denotes more of a private scope, so if you think is confusing I'm fine to change it. 
What would you use in that case?

Comment (rak-phillip) [reply shell/components/KeyValueView.vue:69]: I've encountered the convention of using `_` to denote "private" scope in some of our models and classes, for example:

https://github.com/rancher/dashboard/blob/75c595f97f8cfe1d0c2a37242306863ec4d61486/shell/plugins/dashboard-store/resource-class.js#L1709-L1739

I know that I have encountered it elsewhere throughout Dashboard, and it's a common enough convention in javascript where I think that its meaning is clear.

**Review by rak-phillip — APPROVED** [review]

File: shell/components/KeyValueView.vue (hunk 1 above)
Comment (codyrancher) [reply shell/components/KeyValueView.vue:69]: It was just a question. It's fine to ship as is.

**Pull request merged**